cypress/e2e/tests/pages/extensions/kubewarden.spec.ts

```diff
@@ -7,18 +7,22 @@ import { catchTargetPageException } from '@/cypress/support/utils/exception-util
 
 const extensionName = 'kubewarden';
 
-describe('Kubewarden Extension', { tags: ['@extensions-temp-excluded', '@adminUser'] }, () => {
+describe('Kubewarden Extension', { tags: ['@extensions', '@adminUser'] }, () => {
   before(() => {
     catchTargetPageException('Navigation cancelled');
     cy.login();
 
     const extensionsPo = new ExtensionsPagePo();
+    const kubewardenPo = new KubewardenExtensionPo();
 
     extensionsPo.goTo();
     extensionsPo.waitForPage();
 
     // install the ui-plugin-charts repo
-    extensionsPo.addExtensionsRepository('https://github.com/rancher/ui-plugin-charts', 'main', 'rancher-extensions');
+    kubewardenPo.addChartsRepoIfNeeded();
+
+    // Ensure the catalog cache is up to date before continuing
+    cy.wait(2500); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
We should be able to avoid this by waiting for the repo to become `active` in cypress/e2e/po/pages/extensions.po addExtensionsRepository